lib/config.js

```diff
@@ -118,6 +118,13 @@ var normalizeConfig = function (config, configFilePath) {
   // normalize urlRoot
   config.urlRoot = normalizeUrlRoot(config.urlRoot)
 
+  config.protocol = config.protocol || 'http:'
+  if (['http:', 'https:'].indexOf(config.protocol) === -1) {
```

Comment:
Let's make the values `http` and `https` without the `:`, and strip it, if someone adds it by accident.